### DISCUSSION TITLE
Predicting auto induction effect when fm CYP is low

### DISCUSSION THREAD

**Discussion by NileshGaud** [Open-Systems-Pharmacology/Forum#859] [Questions & Problems] [answered]
Hello,

I would like to predict the auto induction effect for a molecule with fm CYP 0.1 and remaining metabolism is by phase 2 metabolism. Compound is potent inducer similar to rifampicin. 
Will it be possible to simulate auto induction in this case using PKSim? if yes, how it can be done ? I have intrinsic clearance data from hepatocyte.

**Comment by StephanSchaller** [comment, marked as answer]
Dear Nilesh,

yes, this is possible. You will find some instructions and background info here:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-defining-inhibition-induction-processes

Let me know if you have additional questions.

Stephan

**Comment by NileshGaud** [comment]
Thank you for your reply.

I am able to set induction parameters.
But how should I set metabolism of my compound  by cyp3a4 and phase enzyme
together.
As only 10% of my compound is metabolized by cyp.
Do I need to correct the intrinsic Clearance from hepatocyte as it consist
of both phase 1 and phase 2 metabolism.

This is my confusion.

Regards
Nilesh

**Comment by StephanSchaller** [comment]
implement your phase II metabolism also as enzymes and use the output "fraction metabolized" to check the fractional contributions of each pathway/enzyme

**Comment by NileshGaud** [comment]
Hi Stefan,

Can you please elaborate on "implement phase II also as enzymes" . I did not get your point. 

Thank you for your help

**Comment by StephanSchaller** [comment]
Hi @NileshGaud ,

maybe it helps if you map out the mass balance for your compound so we better understand your needs.
But in the end, the "fraction metabolized" output will help you check the contribution of each metabolizing pathway (and also check the amount excreted to feces/urine in case of fecal/renal excretion).

Best,
Stephan

**Comment by NileshGaud** [comment]
Hi Stephan,

I have a compound that gets metabolised by CYP3A4 fm 0.1 and remaining by phase 2 metabolism (conjugation) fm 0.9. There is no parent excreted in feces/urine. 
This compound is potent inducer of CYP3A4 and I am interested in predicting auto-induction effect. 
Questions:
1. What input option should I used in this case to include both metabolism process in the model. Is it posible to include phase 2 metabolism?

Regards,
Nilesh

**Comment by StephanSchaller** [comment]
Hi Nilesh,

assuming 10% metabolism by CYP3A4 and 90% by other enzymes:
1. implement the expression of CYP3A4 and the other enzymes in the individual building block (e.g. load from expression database, see https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data)
2. define enzyme metabolism in the compound building block and select the available (previously in the individual building block defined) enzymes and define the metabolic rate. (https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow)
3. simulate and check "fraction metabolized"
4. adjust metabolism such that fm CYP3A4 = 0.1 and the sum of fm for other enzymes = 0.9

**Comment by NileshGaud** [comment]
Ok Thanks
